lib/component-repository/src/models/ComponentConfig.js

```diff
@@ -5,9 +5,9 @@ const Schema = mongoose.Schema;
 // Define schema
 const componentConfig = new Schema(
   {
-    componentId: {
+    virtualComponentId: {
       type: Schema.Types.ObjectId,
-      required: [true, 'Configuration requires a Component'],
+      required: [true, 'Configuration requires a virtual Component'],
     },
     authClientId: { type: Schema.Types.ObjectId },
     tenant: {
```

Comment:
Check the compound index to be sure that it virtualComponentId and not componentId

Reply:
I think we were talking together when we changed that!

Reply:
yes, but there is an index in this file that reference to componentId that it is wrong I guess, not sure

Reply:
Done